xpdtools/pipelines/raw_pipeline.py

```diff
@@ -216,6 +216,9 @@ def integration(map_res, mask, wavelength, pol_corrected_img, **kwargs):
         pol_corrected_img.map(np.ravel), emit_on=1
     )
 
+    std = f_img_binner.starmap(
```

Comment:
move to independent thing so we don't break the cli (and can have this as an option)